grades/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.forms import inlineformset_factory
+from grades.forms import GradingSystemForm
+from grades.models import GradingSystem, GradeRange
+
+# Create your views here.
+@login_required
+def grading_systems(request):
+    user = request.user
+    default_grading_system = GradingSystem.objects.get(name='default')
```

Comment:
Problem is, all users share the same default model. If one user decides to play around with it, it will affect all users.

Reply:
We are providing default grading system just for a reference, the only thing then we can do is not give permission to edit the default system.